src/App.css

```diff
@@ -30,3 +30,34 @@
     transform: rotate(360deg);
   }
 }
+
+/* Calvin's CSS */
+
+a,
+a:hover,
+a:active,
+a:focus,
+a:visited {
+  color: #222;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+h3 {
+  margin: 0;
+}
+
+label {
+  display: block;
+}
+
+.whiskey-card {
```

Comment:
Yeah, eventually we can start to separate the CSS out to the components it's associated with, like a file called **note.css** with `.whiskey-note {}`, etc.

This is fine for now though.